openfisca_france/model/prestations/complement_are.py

```diff
@@ -0,0 +1,369 @@
+        csg_mensuelle = individu('complement_are_csg_mensuelle', period)
+        crds_mensuelle = individu('complement_are_crds_mensuelle', period)
+
+        return round_(crc_mensuelle + csg_mensuelle + crds_mensuelle, 2)
+
+
+class complement_are_crc(Variable):
+    value_type = float
+    entity = Individu
+    label = 'Montant de Cotisation de Retraite Complémentaire (CRC)'
+    definition_period = MONTH
+    set_input = set_input_divide_by_period
+    reference = [
+        'https://www.unedic.org/indemnisation/fiches-thematiques/cumul-allocation-salaire',
+        'https://www.unedic.org/indemnisation/fiches-thematiques/retenues-sociales-sur-les-allocations'
+        ]
+
+    def formula(individu, period, parameters):
+        allocation_journaliere = individu('allocation_retour_emploi_journaliere', period)
+        salaire_journalier_reference = individu('salaire_journalier_reference_are', period)
+        seuil_exoneration_crc = parameters(period).chomage.complement_are.seuil_exoneration_crc
+        coefficient_crc = parameters(period).chomage.complement_are.coefficient_crc
+
+        return round_(
+            where(
+                allocation_journaliere > seuil_exoneration_crc,
+                salaire_journalier_reference * coefficient_crc,
+                0),
+            2)
+
+
+class complement_are_crc_mensuelle(Variable):
+    value_type = float
+    entity = Individu
+    label = 'Montant de Cotisation de Retraite Complémentaire (CRC)'
+    definition_period = MONTH
+    set_input = set_input_divide_by_period
+    reference = [
+        'https://www.unedic.org/indemnisation/fiches-thematiques/cumul-allocation-salaire',
+        'https://www.unedic.org/indemnisation/fiches-thematiques/retenues-sociales-sur-les-allocations'
+        ]
+
+    def formula(individu, period):
+        complement_are_crc = individu('complement_are_crc', period)
+        complement_are_nombre_jours_indemnisables = individu('complement_are_nombre_jours_indemnisables', period)
+
+        return round_(complement_are_crc * complement_are_nombre_jours_indemnisables, 2)
+
+
+class complement_are_csg(Variable):
+    value_type = float
+    entity = Individu
+    label = 'Montant des Contributions Sociales Généralisées (CSG)'
+    definition_period = MONTH
+    set_input = set_input_divide_by_period
+    reference = [
+        'https://www.unedic.org/indemnisation/fiches-thematiques/cumul-allocation-salaire',
+        'https://www.unedic.org/indemnisation/fiches-thematiques/retenues-sociales-sur-les-allocations'
+        ]
+
+    def formula(individu, period, parameters):
+        allocation_journaliere_brute_are = individu('complement_are_allocation_journaliere_brute_are', period)
+        seuil_exoneration_contributions = parameters(period).chomage.complement_are.seuil_exoneration_contributions
+        coefficient_assiette_contributions = parameters(period).chomage.complement_are.coefficient_assiette_contributions
```

Comment:
En général, regrouper les recherches récurrentes dans l'arborescence des paramètres ? 
Par exemple : 
```suggestion
        parametres_complement_are = parameters(period).chomage.complement_are
        seuil_exoneration_contributions = parametres_complement_are.seuil_exoneration_contributions
        coefficient_assiette_contributions = parametres_complement_are.coefficient_assiette_contributions
```